src/fidesops/service/connectors/query_config.py

```diff
@@ -277,22 +301,23 @@ def generate_query(
         policy: Optional[Policy] = None,
     ) -> Optional[TextClause]:
         """Generate a retrieval query"""
-
-        filtered_data = self.typed_filtered_values(input_data)
+        filtered_data: Dict[str, Any] = self.typed_filtered_values(input_data)
 
         if filtered_data:
             clauses = []
             query_data: Dict[str, Tuple[Any, ...]] = {}
-            formatted_fields = self.format_fields_for_query(self.fields)
+            formatted_fields: List[str] = self.format_fields_for_query(
+                list(self.field_map().keys())
```

Comment:
Am I correct here in reading that `self.field_map().keys()` will be a list of `FieldPath`s? It looks like the key in this dict is a `FieldPath` rather than a string https://github.com/ethyca/fidesops/pull/154/files#diff-f1aba318d05064942a5f135eb61663026a735a60dcd0a2bd93cd629c40c69a02R264

Reply:
` list(self.field_map().keys())` does return a List of FieldPaths, but those are passed into `format_fields_for_query` which turns the list of FieldPaths into a list of strings